data/json/items/comestibles/junkfood.json

```
@@ -1049,6 +1049,26 @@
     "fun": 6,
     "vitamins": [ [ "vitA", 1 ], [ "calcium", 15 ], [ "junk_allergen", 1 ], [ "milk_allergen", 1 ] ]
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "pouch_cheese",
+    "name": "cheese spread pouch",
+    "weight": "10 g",
+    "color": "yellow",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 89,
+    "description": "Heavily processed cheese spread.  Sterilized and packed atmospherically to last long, but it's often the first component of a MRE to go bad.",
```

```suggestion
    "description": "Heavily processed cheese spread.  Sterilized and packed atmospherically to last long.",
```

it's confusing that description says it is the first component to go bad when it's non-perishable.

---

It's just a factoid, it takes years for it to go bad but still, it's the first to go.

---

it would still confuse players nonetheless.